src/pattern-library/components/patterns/prototype/SelectNextPage.tsx

```diff
@@ -1,11 +1,11 @@
 import classnames from 'classnames';
-import { useCallback, useId, useMemo, useState } from 'preact/hooks';
+import { useId, useState } from 'preact/hooks';
 
 import { Link } from '../../../..';
-import { ArrowLeftIcon, ArrowRightIcon } from '../../../../components/icons';
 import type { SelectNextProps } from '../../../../components/input';
-import { IconButton, InputGroup } from '../../../../components/input';
 import SelectNext from '../../../../components/input/SelectNext';
+import SelectNextInInputGroup from '../../../examples/select-next-in-input-group';
+import SelectNextWithManyOptions from '../../../examples/select-next-with-custom-options';
```

Comment:
Explicitly importing some examples to reuse them in some cases where the code snippet is not that important, and it's more important to see how it looks when rendered. This way we don't duplicate that much code.

This could be addressed differently if we had a mechanism to pass props to example files loaded via `exampleFile`, but that is not the case at the moment.